composer.json

```diff
@@ -22,12 +22,15 @@
     "require":      {
         "php":  ">=5.6.0"
     },
+    "conflict": {
```

Comment:
What is hte purpose of adding a `conflict` here? Aren't constraints in `require-dev` enough?

Reply:
This prevents this library to be installed together with incompatible Twig releases. And no, `require-dev` is ignored if you consume this library as a dependency.